vendor/forms.py

```diff
@@ -16,11 +16,27 @@
 
         
 class PriceForm(forms.ModelForm):
-    start_date = forms.DateField(label=_("Start Date"), widget=SelectDateWidget())
-    end_date = forms.DateField(label=_("End Date"), widget=SelectDateWidget())
+    CHOICES = [('not_free', _('Purchase Price')), 
+               ('free', _('Free'))]
+    price_select = forms.ChoiceField(label="", choices=CHOICES, widget=forms.widgets.RadioSelect())
     class Meta:
         model = Price
-        fields = ['cost', 'currency', 'start_date', 'end_date', 'priority']
+        fields = ['price_select', 'cost', 'currency', 'start_date', 'end_date']
+
```

Comment:
would it be easier to set widgets in the meta?

https://docs.djangoproject.com/en/3.1/topics/forms/modelforms/#specifying-widgets-to-use-in-the-form-with-widgets

```
widgets = { 
'cost' : TextInput(attrs={'placeholder':  '##.##',  'class': 'w-50' }
# .... etc
}